actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -200,7 +201,7 @@ def method_missing(called, *args, &block)
       # hash to +options+. Set +escape+ to false to disable attribute value
       # escaping.
       #
-      # ==== Options
+      # ==== Options (Legacy syntax)
```

Comment:
Adding `(Legacy syntax)` to clarify that line 205 below (both symbols and strings) only applies to the legacy syntax, whereas the new syntax only accepts symbols.